skopt/utils.py

```diff
@@ -292,3 +295,50 @@ def cook_estimator(base_estimator, space=None, **kwargs):
 
     base_estimator.set_params(**kwargs)
     return base_estimator
+
+def transform_dims(dimensions, estimator="GP"):
```

Comment:
`normalize_dimensions` is more clear, I think. Would it be possible that in the future the function could handle some special case for any of the estimators not involving normalizing the dimensions? If so, then `transform_dimensions` may be better.

Either way, I agree it could be more clear.